src/components/molecules/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,108 @@
+import Image from "next/image";
+import { Dispatch, SetStateAction } from "react";
+import styles from "./styles/Navbar.module.scss";
+
+const navOptions = [
+  { option: "home", selected: true },
+  { option: "info" },
+  { option: "pc release" },
+  { option: "videos" },
+  { option: "screens" },
+  {
+    option: "features",
+    subOptions: [
+      "the van der linde gang",
+      "the frontier, cities & towns",
+      "wildlife",
+      "weaponry",
+      "music",
+      "social club",
+    ],
+  },
+  { option: "news" },
+  { option: "online" },
+  { option: "downloads" },
+  { option: "buy now" },
+];
+
+type NavProps = {
+  show: boolean;
+  setShowNavbar: Dispatch<SetStateAction<boolean>>;
+};
+
+const Navbar: React.FC<NavProps> = ({ show, setShowNavbar }) => {
+  if (!show) return <></>;
+
+  return (
+    <>
+      <div
+        className={styles.overlay}
+        onClick={() => setShowNavbar((s) => !s)}
+      />
+      <nav className={styles.container}>
+        <div className={styles.background_img} />
+        <div className={styles.bullets_container}>
+          {navOptions.map((no) => {
```

Comment:
```suggestion
          {navOptions.map((navOption) => {
```
Acho que assim fica mais legivel no primeiro contato.